en/messages.json

```diff
@@ -1,6 +1,6 @@
 {
-    "extensionDescription": {
-        "message": "Firefox Relay makes it easy to create email aliases that forward to your real inbox. Use it to protect your online accounts from hackers and unwanted messages.",
+    "extensionDescription_v1": {
```

Comment:
main repo needs updating to match this key too.

Reply:
Done: https://github.com/mozilla/fx-private-relay-add-on/pull/249/commits/bfbc2d8ce775bcc1c06cf255205f9886fb86272a